intranet/apps/eighth/serializers.py

```diff
@@ -93,7 +93,13 @@ class EighthBlockDetailSerializer(serializers.Serializer):
     comments = serializers.CharField(max_length=100)
 
     def process_scheduled_activity(
-        self, scheduled_activity, request=None, user=None, favorited_activities=None, recommended_activities=None, available_restricted_acts=None
+        self,
+        scheduled_activity,
+        request=None,
+        user=None,
+        favorited_activities=None,
+        recommended_activities=None,  # pylint: disable=unused-argument
```

Comment:
Isn't this argument unused?

Reply:
I removed the only use in ea97f5c0f4ac3b4d1fefb1da8dadc983a04159cc

Reply:
Oops, that's actually correct. I was looking at an older version when I made the change.

I'll try broader changes to remove the variable further up.